auth/app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use Laravel\Lumen\Routing\Controller as BaseController;
+use Lcobucci\JWT\Builder;
+use Lcobucci\JWT\Parser;
+use Lcobucci\JWT\Signer\Hmac\Sha256;
+use Miner\Auth\Entities\User;
+use Miner\Auth\Repositories\UserRepository;
+use Miner\Auth\Entities\Uuid;
+
+class AuthController extends BaseController
+{
+    public function login(Request $request, UserRepository $userRepository, Cache $cache)
+    {
+        $this->validate($request, [
+            'username' => 'required',
+            'password' => 'required'
+        ]);
+
+        $email = $request->get('email');
```

Comment:
Since we are reaching to the request for the email here, should we not require it in validation rules?